.travis.yml

```diff
@@ -21,25 +21,6 @@ matrix:
         - LLVM_CONFIG=llvm-config-7
         - DEPLOY=0
 
-    - os: linux
```

Comment:
Is it expected to remove this build ?

Reply:
I've moved it at the end (and replaced rust stable with rust nightly like all the other builds except one), to put it closer to the other build which is similar to this one.
Not really needed, just a small change.